app/src/main/java/com/github/fribourgsdp/radio/FireBaseTestActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.fribourgsdp.radio
+
+import android.content.ContentValues
+import android.content.ContentValues.TAG
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.TextView
+import com.google.firebase.firestore.ktx.firestore
+import com.google.firebase.ktx.Firebase
+
+class FireBaseTestActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_fire_base_test)
+
+
+        val db = Database()
+
+        val userAuthUID = "m0sd9l"//FirebaseAuth.getInstance().currentUser?.uid
```

Comment:
Maybe you should remove the old commented code.

Reply:
I added a clarification. This code will be used later.